packages/cloudi/cloudi.2.0.7/opam

```diff
@@ -0,0 +1,21 @@
+opam-version: "2.0"
+maintainer: "Michael Truog <mjtruog at protonmail dot com>"
+authors: "Michael Truog <mjtruog at protonmail dot com>"
+homepage: "https://cloudi.org/"
+bug-reports: "https://github.com/CloudI/cloudi_api_ocaml/issues"
+license: "MIT"
+dev-repo: "git+https://github.com/CloudI/cloudi_api_ocaml.git"
+build: [
+  [make]
+]
+depends: [
+  "ocaml" {>= "4.03.0"}
+  "base-unix"
+  "zarith" {>= "1.13"}
```

Comment:
In this case you don't really need this line

Reply:
While it might not be strictly necessary, I wanted to be transparent about the dependency on zarith, so it appears in the cloudi package information.

Reply:
Thanks for the clarification